imaginedsf-custom-theme/acf-json/group_5d5c3f52f1b77.json

```diff
@@ -0,0 +1,98 @@
+{
+    "key": "group_5d5c3f52f1b77",
+    "title": "Basemaps",
```

Comment:
Same comment here as on Proposals: no title necessary, and the link should be a Post Object field type filtered to Maps.

Reply:
Made both changes, and added a required 'Year' field to Basemaps (forgot to do it the first time) to keep it consistent with Proposals.